core/server/master/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -739,20 +742,13 @@ public FileInfo getFileInfo(AlluxioURI path)
       throws FileDoesNotExistException, InvalidPathException, AccessControlException {
     Metrics.GET_FILE_INFO_OPS.inc();
 
-    // Get a READ lock first to see if we need to load metadata, note that this assumes load
-    // metadata for direct children is disabled by default.
-    try (LockedInodePath inodePath = mInodeTree.lockInodePath(path, InodeTree.LockMode.READ)) {
+    try (JournalContext journalContext = createJournalContext();
+        LockedInodePath inodePath = mInodeTree.lockInodePath(path, InodeTree.LockMode.READ)) {
       mPermissionChecker.checkPermission(Mode.Bits.READ, inodePath);
       if (inodePath.fullPathExists()) {
         // The file already exists, so metadata does not need to be loaded.
         return getFileInfoInternal(inodePath);
       }
-    }
-
-    try (JournalContext journalContext = createJournalContext();
-        LockedInodePath inodePath = mInodeTree.lockInodePath(path, InodeTree.LockMode.WRITE)) {
```

Comment:
`getFileInfo` is a read operation, and the reason we used the WRITE lock before was to allow loading metadata. With the new creation locking scheme, we no longer have to use the WRITE lock mode, since the new scheme will take care of creating inodes along the path with the correct lock modes.